lib/ebooks/ia_searcher.rb

```diff
@@ -15,7 +15,8 @@ def self.search(query, per_page, page)
       response = conn.get do |req|
         req.url '/advancedsearch.php'
         req.params['q'] = query
-        req.params['fl'] = IA_FIELDS
+        # the search API expects fl and sort to be arrays
```

Comment:
```suggestion
        # the search API supports comma-delimited or array fl values, but since the form at
        # https://archive.org/advancedsearch.php uses arrays we'll follow that convention
```